src/shaping/shaping_linux.go

```diff
@@ -301,7 +332,7 @@ func setupRootQdisc(link_name string) error {
 	}
 	for _, q := range qdiscs {
 		if err := netlink.QdiscDel(q); err != nil {
-			return fmt.Errorf("Could not delete qdisc: %v", err)
+			Log.Println("warning: Could not delete root qdisc:", err)
```

Comment:
Can you add this comment in the code under a `FIXME` so we remember why this is done like this.